src/API/Plugins.php

```diff
@@ -220,6 +220,12 @@ public function install_plugin( $request ) {
 	public function install_plugins( $request ) {
 		$plugins = explode( ',', $request['plugins'] );
 
+		// If both Jetpack and WooCommerce Payments are being installed, Jetpack must be installed first.
+		if ( in_array( 'jetpack', $plugins, true ) && in_array( 'woocommerce-payments', $plugins, true ) ) {
```

Comment:
What do you think about introducing a hook here to allow filtering of plugins?  (Naming of this filter could probably be improved)

`$plugins = apply_filters( 'woocommerce_admin_plugins_pre_install', $plugins )`

Then we can create a separate function to handle this install error.